core/src/main/java/org/camunda/spin/json/SpinJsonNode.java

```diff
@@ -45,11 +45,22 @@
    * Fetches the last index of the searched object in an array.
    *
    * @param searchObject Object for which the index should be searched.
-   * @return {@link Integer} index of searchObject or -1 if object not found.
+   * @return {@link Integer} index of searchObject.
    * @throws SpinJsonException if the current node is not an array.
+   * @throws SpinJsonPropertyException if object is not found.
    */
   public abstract Integer lastIndexOf(Object searchObject);
 
+  /**
+   * Fetches the last index of the searched object in an array.
```

Comment:
Please adjust the documentation here. It seems to be a copy of the `lastOndexOf` method.